devnet-2/seeds.txt

```diff
@@ -0,0 +1 @@
+74c0c793db07edd9b9ec17b076cea1a02dca511f@46.101.28.34:26656
```

Comment:
Orthogonal to this PR but probably better to have a list 3-4 reliable seeds IMO.

Ideally, we would run a few reliable peers in seed mode only: https://docs.tendermint.com/master/spec/p2p/config.html#seed-mode

and then add their IDs here.